plugins/tag.py

```diff
@@ -183,8 +186,27 @@ def tags(inp, chan='', db=None):
     '.tags <nick>/list -- get list of tags for <nick>, or a list of tags {related: .tag, .untag, .tagged, .is}'
```

Comment:
`'.tags <nick>/<nick1> <nick2> .../list -- get list of tags for <nick>, tags shared by <nicks>, or a list of tags {related: .tag, .untag, .tagged, .is}'`